### PR TITLE
Feat/edco 6

### PR DESCRIPTION
**Pull request by leleOFA**

### CODE REVIEW

File: src/stories/Feature/Feature.stories.tsx
Comment (mmoio) [inline src/stories/Feature/Feature.stories.tsx:1]: You should remove this file if all of its stories are in the dark.stories.tsx and light.stories.tsx

File: src/stories/Feature/dark.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  },
+  {
+    icon: <AccountBoxIcon />,
+    title: 'Immediato',
+    subtitle:
+      "I tempi di attesa tra l'invio e la ricezione delle notifiche spariscono",
+  },
+];
+export default {
+  /* 👇 The title prop is optional.
```
Comment (mmoio) [inline src/stories/Feature/dark.stories.tsx:50]: Remove this comment

File: src/stories/Feature/dark.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+   * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
+  title: 'Feature',
```
Comment (mmoio) [inline src/stories/Feature/dark.stories.tsx:54]: Title should be 'Feature/dark'

File: src/stories/Feature/light.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  },
+  {
+    icon: <AccountBoxIcon />,
+    title: 'Immediato',
+    subtitle:
+      "I tempi di attesa tra l'invio e la ricezione delle notifiche spariscono",
+  },
+];
+export default {
+  /* 👇 The title prop is optional.
```
Comment (mmoio) [inline src/stories/Feature/light.stories.tsx:50]: Remove this comment

File: src/stories/Feature/light.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+   * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
+  title: 'Feature',
```
Comment (mmoio) [inline src/stories/Feature/light.stories.tsx:54]: Title should be 'Feature/light'

**Review by mmoio — CHANGES_REQUESTED** [review]

File: src/stories/Feature/Feature.stories.tsx
Comment (leleOFA) [reply src/stories/Feature/Feature.stories.tsx:1]: do it

File: src/stories/Feature/dark.stories.tsx (hunk 1 above)
Comment (leleOFA) [reply src/stories/Feature/dark.stories.tsx]: do it

File: src/stories/Feature/light.stories.tsx (hunk 3 above)
Comment (leleOFA) [reply src/stories/Feature/light.stories.tsx]: do it

**Review by mmoio — APPROVED** [review]
LGTM

**Review by vito80ba — APPROVED** [review]

**Pull request merged**